api/notd/store/saver.py

```diff
@@ -98,7 +96,7 @@ async def update_token_metadata_item(self, tokenMetadataId: int, metadataUrl: Op
             values[TokenMetadataTable.c.description.key] = description
         if name != _EMPTY_STRING:
             values[TokenMetadataTable.c.name.key] = name
-        if attributes != _EMPTY_OBJECT:
+        if attributes is not _EMPTY_OBJECT:
```

Comment:
this looks wierd

Reply:
I should change it back to != ?

Reply:
i think so - its only common for None and booleans to be checked with `is` and `is not`